app/components/Onboarding/BtcPayServer.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './BtcPayServer.scss'
+
+const BtcPayServer = ({
+  connectionString,
+  connectionStringIsValid,
+  setConnectionString,
+  startLndHostError
+}) => (
+  <div className={styles.container}>
+    <section className={styles.input}>
+      <label htmlFor="connectionString">Connection String:</label>
+      <textarea
+        type="text"
+        id="connectionString"
+        rows="9"
+        placeholder="BTCPayServer Connection String"
+        className={`${
```

Comment:
Quoting looks unnecessary here

Reply:
Could you elaborate please @Empact . I'm not sure I follow

Reply:
You should be able to remove `\`${` bit, and the closing, because then it would resolve down to `undefined` or `styles.error` directly, which don't need quoting afaict.

Reply:
Gotcha. You are right - I've altered that. Thanks.